runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/fn/control/RegisterAndProcessBundleOperationTest.java

```diff
@@ -242,200 +240,6 @@ public void close() {}
-                                                                  .setValue(finalCounterValue)))
-                                                  .build())))
-                          .build();
-                    });
-              case PROCESS_BUNDLE_PROGRESS:
-                return CompletableFuture.completedFuture(
-                    responseFor(request)
-                        .setProcessBundleProgress(
-                            BeamFnApi.ProcessBundleProgressResponse.newBuilder()
-                                .setMetrics(
-                                    BeamFnApi.Metrics.newBuilder()
-                                        .putPtransforms(
-                                            stepName,
-                                            BeamFnApi.Metrics.PTransform.newBuilder()
-                                                .addUser(
-                                                    BeamFnApi.Metrics.User.newBuilder()
-                                                        .setMetricName(metricName)
-                                                        .setCounterData(
-                                                            BeamFnApi.Metrics.User.CounterData
-                                                                .newBuilder()
-                                                                .setValue(counterValue)))
-                                                .build())))
-                        .build());
-              default:
-                // block forever
-                return new CompletableFuture<>();
-            }
-          }
-
-          @Override
-          public void close() {}
-        };
-
-    RegisterAndProcessBundleOperation operation =
-        new RegisterAndProcessBundleOperation(
-            idGenerator,
-            instructionRequestHandler,
-            mockBeamFnStateDelegator,
-            REGISTER_REQUEST,
-            ImmutableMap.of(),
-            ImmutableMap.of(),
-            ImmutableMap.of(),
-            ImmutableTable.of(),
-            ImmutableMap.of(),
-            mockContext);
-
-    operation.start();
-
-    // Force some intermediate metrics to test crosstalk is not introduced
-    BeamFnApi.Metrics metrics = MoreFutures.get(operation.getProcessBundleProgress()).getMetrics();
-    BeamFnApi.Metrics.User userMetric = metrics.getPtransformsOrThrow(stepName).getUser(0);
-    assertThat(userMetric.getMetricName(), equalTo(metricName));
-    assertThat(userMetric.getCounterData().getValue(), not(equalTo(finalCounterValue)));
-
-    processBundleLatch.countDown();
-    operation.finish();
-
-    metrics = MoreFutures.get(operation.getFinalMetrics());
-
-    userMetric = metrics.getPtransformsOrThrow(stepName).getUser(0);
-    assertThat(userMetric.getMetricName(), equalTo(metricName));
-    assertThat(userMetric.getCounterData().getValue(), equalTo(finalCounterValue));
-  }
```

Comment:
You way want to add this back, if you truly cannot yet delete the deprecated metrics

Reply:
Which deprecated metrics are you referring to?

Reply:
The legacy metrics proto deleted in this PR. You may want to check with to make sure no other runner is consuming this.